resources/web/docs.js

```diff
@@ -303,8 +317,14 @@ jQuery(function() {
           curlText += comment + '\n';
         } else {
           path = path.replace(/^\//, '').replace(/\s+$/,'');
-          path += path.includes('?') ? '&pretty' : '?pretty';
-          curlText += 'curl -X' + method + " 'localhost:9200/" + path + "'";
```

Comment:
`localhost:9200` was hardcoded here. I updated it to include the host/port configured through the settings.